strum_macros/src/macros/strings/display.rs

```diff
@@ -124,7 +124,17 @@ pub fn display_inner(ast: &DeriveInput) -> syn::Result<TokenStream> {
                         }
                     }
                 }
-                Fields::Unit => quote! { #name::#ident #params => ::core::fmt::Display::fmt(&format!(#output), f) }
+                Fields::Unit => {
```

Comment:
Related to #363 
Since `format!` is no longer used for `Unit`, please do not check the curly brackets on it.
This will cause the use of a single curly bracket to fail to compile, and two curly brackets cannot be escaped.
Or continue to use `format!` for `Unit`, and release it as a breaking change.